docs/pages/example/cluster-html.js

```diff
@@ -0,0 +1,10 @@
+/*---
+title: Display HTML clusters with custom properties
+description: An advanced example of using Mapbox GL JS clustering with HTML markers and custom property expressions.
```

Comment:
Outlining the general strategy that the example takes would be very useful here to set context for the code.

Reply:
Can you elaborate? Even the general strategy would need description that seems too long for the purpose.

Reply:
Maybe it can be added in the code section then?

Strategy would Outline that this example is using clustered GeoJSON with filtered layers and continuously updating Markers.

Reply:
Right — I've extended the description.